inc/modules/export/xhtml/class-xhtml11.php

```diff
@@ -419,7 +420,7 @@ function moveFootnotes() {
 
 		$my_get = $_GET;
 		unset( $my_get['timestamp'], $my_get['hashkey'] );
-		$cache = get_transient( self::TRANSIENT );
+		$cache = false;
```

Comment:
just a reminder bro, we should remember replace this once the PR is ready. I use to forget it.